### ISSUE TITLE
Problem with upgrading to v2.11

### ISSUE THREAD

**Issue by Asw1n** [SignalK/signalk-server#1801]
Upgrading the server from within the server GUI to version 2.11 fails under Windows. The first sympton is that the server won't reboot. Also the start-server.cmd has gone missing. I think the reason is that a new version for nodejs is needed and is not installed.

I got the server up and running again by deleting the nmn subdirectory and running a windows install.

The windows installer warns for this condition. When upgrading from within the server GUI no warning is issued and the upgrade continues compromising the software.

**Comment by dirkwa** [comment]
I hope this issue can be closed by now or is it still a topic? 
v2.23.0 is out.

**Comment by Asw1n** [comment]
I got the same problem when upgrading from 2.22 to 2.23.

**Comment by tkurki** [comment]
@Asw1n I changed the title, does that reflect accurately the problem?

If it does then the Windows install should set env variable SIGNALK_DISABLE_SERVER_UPDATES. @cmotelet thoughts?

**Comment by Asw1n** [comment]
It does

**Comment by cmotelet** [comment]
No, the Windows installer doesn't set the SIGNALK_DISABLE_SERVER_UPDATES variable in fact, it does the opposite: SIGNALK_SERVER_IS_UPDATABLE=1
As I posted on Discord, the update works via the GUI:
https://discord.com/channels/1170433917761892493/1482370313684516946
